cfme/tests/services/test_iso_service_catalogs.py

```diff
@@ -33,6 +34,9 @@ def pytest_generate_tests(metafunc):
             # No provisioning data available
             continue
 
+        if args['provider_type'] == "scvmm":
```

Comment:
What about VMware? I don't think they support ISO either do they? So maybe we should flip it and say ```if not rhevm```